src/platform/logging/index.ts

```diff
@@ -49,25 +49,25 @@ export function traceLog(message: string, ...args: Arguments): void {
 }
 
 export function traceError(message: string, ...args: Arguments): void {
-    if (globalLoggingLevel >= LogLevel.Error) {
+    if (globalLoggingLevel <= LogLevel.Error) {
         loggers.forEach((l) => l.traceError(message, ...args));
     }
 }
 
 export function traceWarning(message: string, ...args: Arguments): void {
-    if (globalLoggingLevel >= LogLevel.Warn) {
+    if (globalLoggingLevel <= LogLevel.Warn) {
```

Comment:
Not sure why this was flipped.
Basically the order is Error=40, Warn, ... info, debug, trace=5.
The higher the number, the lower the trace level. I know this is confusing, however this is how it was done and I believe somethign we might want to change. Info & debug are the same. We don't log anything to `debug`, when we use traceinfo we log using the `Info` level and when we use traceVerbose we log using the `trace` level

I'm reverting as I'm assuming all of these changes were unintended